client/state/selectors/get-post-share-published-actions.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { enrichPublicizeActionsWithConnections } from 'state/selectors/utils/';
+
+/**
+ * Return a share-published-actions array propagaring data from publicize connections.
+ *
+ * @param {Object} state Global state tree
+ * @param {Number} siteId Site ID
+ * @param {Number} postId Shared Post ID
+ * @return {Array} share scheduled actions array
+ */
+export default function getPostSharePublishedActions( state, siteId, postId ) {
```

Comment:
We totally lost control over this work and we need to merge in master whatever is not hurting previous flow. I'll make sure we write issue for this.